src/geojson.rs

```diff
@@ -413,7 +411,7 @@ impl<'de, T: GeoJsonGeometry<f64> + PointT + Clone> Deserialize<'de> for MultiPo
     }
 }
 
-impl<T: GeoJsonGeometry<f64> + PointT + Clone + Serialize> Serialize for GeometryContainer<T> {
```

Comment:
I thought Clone was required for something and some features may not work without it, is it safe to remove it?